app/jobs/seed_users_data_job.rb

```diff
@@ -45,7 +45,7 @@ def patient_traits
     @patient_traits ||=
       {
         registered_patient: {
-          time_fn: -> { Faker::Time.between(from: 9.month.ago, to: Time.now) },
+          time_fn: -> { Faker::Time.between(from: 24.months.ago, to: Time.now) },
```

Comment:
It may be worth extracting this to a constant - `SEED_DATA_AGE` or something like that. Then we could easily expand or contract this for everything that needs to go back n months.